src/Route.php

```diff
@@ -5,12 +5,12 @@
 
 use function preg_match;
 
-class Route
+class Route implements IRoute
 {
     /** @var string */
     public $httpMethod;
 
-    /** @var string */
+    /** @var ?string */
```

Comment:
I think this is incorrect (since the constructor disallow `null` here)

Reply:
Good catch